deployment/ccip/changeset/globals/ocr3.go

```diff
@@ -52,26 +52,25 @@ var (
 	CommitOCRParamsForEthereum = withOverrides(
 		CommitOCRParams,
 		types.OCRParameters{
-			DeltaRound: 90 * time.Second,
-			DeltaStage: 60 * time.Second,
```

Comment:
Removed deltaStage since it doesn't get used by the plugin, only TransmissionDelayMultiplier gets used.